xdsl/dialects/varith.py

```diff
@@ -67,10 +69,38 @@ class VarithMulOp(VarithOp):
     name = "varith.mul"
 
 
+@irdl_op_definition
+class VarithSelectOp(IRDLOperation):
+    """
+    Variadic selection operation
+    """
```

Comment:
Can you add a tiny bit more info here what the semantics are? Especially if the index is out of range?

Reply:
I've added to the docstring but am not certain that I am happy with the undefined behaviour